Gems/PhysX/Code/Source/EditorRigidBodyComponent.cpp

```diff
@@ -375,6 +375,14 @@ namespace PhysX
         InitPhysicsTickHandler();
     }
 
+    EditorRigidBodyComponent::EditorRigidBodyComponent(
+        const EditorRigidBodyConfiguration& configuration, const RigidBodyConfiguration& physxSpecificConfiguration)
```

Comment:
question: Is the parameter name correct here? `physxSpecificConfiguration` (maybe it's not as clear as we're inside the `PhysX` namespace, just wanted to double check

Reply:
Maybe to clarify we should rename `configuration` to `physxEditorSpecificConfiguration`? Though I know that's a bit of a mouthful 😅

Reply:
By the way, this also enables us to set solver iterations for the RigidBody when creating the component - an issue we were about to raise. Great to see it here!

Reply:
`physxSpecificConfiguration` is the parameter we use in RigidBodyComponent, so jus trying to be consistent. 

`physxEditorSpecificConfiguration` may be a bit misleading since pretty much all parameters in there are non-editor and non-physx specific ones. 

We have quite a few configuration structs now. Would be nice to consolidate them at some point..